### PR TITLE
Feature/pattern scaffolding scss

### PR DESCRIPTION
**Pull request by Tregonia**

### CODE REVIEW

**Comment by Tregonia** [comment]
https://github.com/mediacurrent/theme_generator_8/issues/114

**Review by mariohernandez — CHANGES_REQUESTED** [review]
Although there might be some useful things in this PR, I don't feel we need to maintain code that is part of Pattern Lab.  Pattern Lab is constantly changing and don't think linting its code or doing other code updates is the best use of our time.  Let's discuss with the team to determine what makes sense to keep from this PR.

**Comment by Tregonia** [comment]
I do not understand what you are referring to with the linting. There is 1 scss file at the location that the gulp tasks. This can be expended on, but there is only 1. The contents of this css file are for style deviations we add to our styleguides. For example, the reason our icon component looks the way it does, is because the styles in this file. 

> Although there might be some useful things in this PR, I don't feel we need to maintain code that is part of Pattern Lab. Pattern Lab is constantly changing and don't think linting its code or doing other code updates is the best use of our time. Let's discuss with the team to determine what makes sense to keep from this PR.

File: generators/app/templates/_gulp-tasks/compress.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = {
   // Compress svg/png/jpg files.
   compressAssets: function() {
     return src([
-      './src/patterns/{global,layout,components}/**/*{.png,.jpg,.svg}'
+      './src/patterns/{global,layout,components}/**/*{.gif,.png,.jpg,.svg}'
```
Comment (rlahoda) [inline generators/app/templates/_gulp-tasks/compress.js:15]: This is fine, can you add .jpeg files as well? that was an update I had planned on making after running into that issue previously.

File: generators/app/templates/_gulpfile.js [hunk 2]
```diff
@@ -118,10 +121,10 @@ function watchFiles() {
 
   // Watch all my images and SVG files and compile if a file changes.
   watch(
-    './src/patterns/**/**/*{.png,.jpg,.svg}',
+    './src/patterns/**/**/*{.gif,.jpg,.png,.svg}',
```
Comment (rlahoda) [inline generators/app/templates/_gulpfile.js:124]: Same as my comment above about adding .jpeg

**Review by rlahoda — APPROVED** [review]
I think adding this in makes a lot more sense than other possible options for overriding styles in the patternlab display. In another project, we're adding in CSS overrides in the `_head.twig` file in a `<style>` tag so adding this into the compiled styles seems like a good idea to consolidate and better track the styles.

I'm approving this with a couple very minor tweaks that aren't required but would be helpful.

**Comment by rlahoda** [comment]
Also, I changed the base for the PR to develop instead of `main`. To better track what's getting added to NPM, it makes sense to only merge into `main` when the code is getting release on NPM as well.

**Review by mariohernandez — APPROVED** [review]

File: generators/app/templates/_gulp-tasks/compress.js (hunk 1 above)
Comment (Tregonia) [reply generators/app/templates/_gulp-tasks/compress.js:15]: We can add that. I wonder tho, that is 5 items at this point. Is there a better way to represent this; aside from more commas?

**Pull request merged**